Problem 2

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <string.h>
+#include <stdlib.h>
+
+using namespace std;
+
+int missingLetter(char text[])
+{
+int variable = (int)text[0];
```

Comment:
I don't think 'variable' is the best name for this 😄

Reply:
Maybe ascii_code?